Include TableQA functionality with TAPAS (#257)
primeqa/primeqa: primeqa/mrc/README.md

eval_samples = 18670
```

>  - PrimeQA also supports answering questions to which answers are collective e.g. lists.

Reviewer: @vishwajeetkumar93 This line needs to be added back in. Thanks!

xlm-roberta-large -> PrimeQA/tydiqa-primary-task-xlm-roberta-large -> NQ Lists: Minimal F1 = 57.44
```

- PrimeQA also supports answering questions over tables
For training and evaluation of a Table Question Answering model on wikisql dataset run the following script:
```shell
       python primeqa/mrc/run_mrc.py --modality "table" \
       --dataset_name "wikisql" \
       --tqa_args_file "primeqa/tableqa/tqa_args.json" \
       --output_dir "models/wikisql/" \
       --model_name_or_path "google/tapas-base" \
       --do_train \
       --do_eval
```

The current performance on wikisql dev set is:
```shell
***** eval metrics *****
Eval denotation accuracy: 86.78%

```

> You can also train the tableqa model on your own custom data. Train the TableQA model on custom data using following script:

Reviewer: ```suggestion
You can also train the tableqa model on your own custom data. Train the TableQA model on custom data using the following script:
```

```shell

>  python primeqa/mrc/run_mrc.py --modality "table" \

Reviewer: No need to show the full command again, just the two new parameters

       --dataset_name "wikisql" \
       --tqa_args_file "primeqa/tableqa/tqa_args.json" \
       --output_dir "models/wikisql/" \
       --model_name_or_path "google/tapas-base" \
       --train_file "<path_to_train.tsv file" \
       --eval_file "<path_to_eval.tsv file" \
       --do_train \
       --do_eval

```

The format of dataset required for training and evaluation is:

`Question_id\tquestion\ttable_path\tanswer_coordinates\tanswer_text`    

The tables in csv format should be placed under `data_path_root/tables/`. The tables should have first row as column headers.

> Refer to [notebooks](../notebooks/tableqa/tableqa_inference.ipynb) for knowing about how to test the pre-trained model available [here](https://huggingface.co/PrimeQA/tapas-based-tableqa-wikisql-lookup).

Reviewer: ```suggestion
Our python [notebook](../notebooks/tableqa/tableqa_inference.ipynb) shows how to test the pre-trained model available [here](https://huggingface.co/PrimeQA/tapas-based-tableqa-wikisql-lookup).
```
